### PR TITLE
Add dockerfile for DockerHub

### PR DESCRIPTION
**Pull request by chraibi**

### CODE REVIEW

File: container/dockerhub/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                       make \
+                       libboost-dev \
+                       libcgal-dev \
+                       clang-format-8 \
+                       python3 \
+                       python3-pip \
+                       python \
+                       python-pip
+
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6084F3CF814B57C1CF12EFD515CF4D18AF4F7421 && \
```
Comment (zeroset) [inline container/dockerhub/Dockerfile]: Do you need clang-format-9?

File: container/dockerhub/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+RUN apt-get update && apt-get install -y software-properties-common
+
+RUN apt-get update && apt-get install -y wget \
+                       git \
+                       g++-8 \
+                       cmake \
+                       make \
+                       libboost-dev \
+                       libcgal-dev \
+                       clang-format-8 \
```
[the remark this answers is not in the document]
Comment (chraibi) [reply container/dockerhub/Dockerfile]: no

File: container/dockerhub/Dockerfile (hunk 1 above)
Comment (chraibi) [reply container/dockerhub/Dockerfile]: no

File: container/dockerhub/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+RUN apt-get update && apt-get install -y software-properties-common
```
Comment (zeroset) [inline container/dockerhub/Dockerfile]: I think this was required for adding the other repository for clang-format 9 and should be removed as well.

File: container/dockerhub/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    && git clone --depth=1 https://github.com/JuPedSim/jpscore.git \
+    && cp jpscore/scripts/setup-deps.sh /opt \
+    && cd /opt && CXX=/usr/bin/g++-8 CC=/usr/bin/gcc-8 ./setup-deps.sh
+
+# build jpscore and jpsreport
+RUN cd /home/jupedsim/workspace/jpscore \
+    && mkdir -p build \
+    && cd build \
+    && cmake  ..\
+    && make
```
Comment (zeroset) [inline container/dockerhub/Dockerfile]: Maybe you should use multithreading here `-j$(nproc)` ?

File: container/dockerhub/Dockerfile (hunk 3 above)
Comment (chraibi) [reply container/dockerhub/Dockerfile]: ok

File: container/dockerhub/Dockerfile (hunk 4 above)
Comment (chraibi) [reply container/dockerhub/Dockerfile]: Yes, thanks!

**Comment by Ozaq** [comment]
I don't think this is the right solution, if i understand this correctly this is supposed to be a usage container not a build container.

As a user I would be quite surprised (annoyed) if I had to download such a fat container. I think the image should not build the software but instead just copy it in from TC. Something like:

```
FROM ubuntu:18:04
COPY <jps executable>
...
```

**Pull request closed without merging**